packages/facet-filter/src/FacetFilterController.js

```diff
@@ -123,7 +123,7 @@ const FacetFilterController = (props) => {
    * Generate facet sections state
    *
    */
-  const displayFacets = facetsConfig.filter((facet) => facet.show).slice(0, 16);
+  const displayFacets = facetsConfig.filter((facet) => facet.show);
```

Comment:
Hi @rana22, Is this specific to ICDC? as removing this one might impact the wider amount of projects.